docs/REST API/rest-api.md

```diff
@@ -8696,6 +8709,7 @@ Get the results of a software installation.
    "host_display_name": "Marko's MacBook Pro",
    "status": "failed",
    "output": "Installing software...\nError: The operation can’t be completed because the item “Falcon” is in use.",
+   "detail": "The install script finished with exit code 0 but Fleet couldn't extract the version from the FalconSensor-6.44.pkg package.",
```

Comment:
With the "version restriction" now added when uploading installers this error code ("Fleet couldn't extract the version from the FalconSensor-6.44.pkg package") won't happen.

Reply:
Ah, good catch. I updated this error message to the error message when Fleet has retried once already:

Failed to verify: The install script finished with exit code 0 but the software is now missing. Fleet already tried to reinstall once.